script.js

```diff
@@ -216,3 +216,65 @@ function createDom() {
 window.addEventListener('load', () => {
   createDom();
 });
+
+function showMessage(input, message, type) {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+  input.className = type ? 'success' : 'error';
+  return type;
+}
+
+function showError(input, message) {
+  return showMessage(input, message, false);
+}
+
+function showSuccess(input) {
+  return showMessage(input, '', true);
+}
+
+function hasValue(input, message) {
+  if (input.value.trim() === '') {
+    return showError(input, message);
+  }
+  return showSuccess(input);
+}
+
+function isLowerCase(input, message) {
+  const email = input.value.trim();
+  if (email !== email.toLowerCase()) {
+    return showError(input, message);
+  }
+  return showSuccess(input);
+}
+
+function validateEmail(input, requiredMsg, invalidMsg, lowerCaseMsg) {
+  if (!hasValue(input, requiredMsg)) {
+    return false;
+  }
+
+  if (!isLowerCase(input, lowerCaseMsg)) {
+    return false;
+  }
+
+  const emailReg = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  const email = input.value.trim();
+  if (!emailReg.test(email)) {
+    return showError(input, invalidMsg);
+  }
+  return true;
+}
+
+const form = document.querySelector('form');
+
+const EmailRequired = 'Please enter your email';
+const EmailInvalid = 'Please enter a correct email address format: name@example.com';
+const EmailLowerCase = 'Please make sure all the letters of your email are in Lowercase';
```

Comment:
- [ ] `required`
- [ ] Good job so far 👏 
- [ ]  as the stated in the requirements.
- [ ] If the validation is not OK, you show an error message to the user `near the submit button` informing them of the error and the form is not sent.
- [ ]  kindly move the error message near the submit button. 🥂